src/controller/org.controller/org.controller.js

```diff
@@ -50,6 +51,52 @@ async function getCna (req, res) {
     })
 }
 
+// Get details on ID quota for a org with the specified org shortname
+async function getOrgIdQuota (req, res) {
+  const shortName = req.params.shortname
+
```

Comment:
Include check that returns 403 if the requestor is not the secretariat or the owning cna